workshops/interactive_maps/README.md

```diff
@@ -0,0 +1,279 @@
+- I recommend going through https://www.w3schools.com/js/js_functions.asp for further clarification with the JS part
+
+
+### 3 - Designing the image 
+Any suitable image will work, but I have chosen an animated isometric map. The image was of the JPG format, but I removed the background to get a png image. Then from that one image, I selected specific areas and created 5 different images, each with a different area highlighted in red.
+For a visual understanding, here is the original image followed by an edited one.
+
+![image 1](https://cloud-fyde5ir0a.vercel.app/image.png)
+![image 2](https://cloud-jmd1dy3k8.vercel.app/image.png)
+
+### 4 - Coding it up
+##### The boilerplate
+Head over to https://repl.it. Creating an account is of personal preference, be warned that you may lose your work if you don't have an account. Click on start coding and search for HTML, CSS, JS. Once you create a repl, you'll end up at this page.
+
+![html boilerplate](https://cloud-5egibvgdv.vercel.app/image.png)
+
+As you can see, repl has already created the boilerplate code for your project. Let's take a quick look. 
+```<!DOCTYPE html>```  is a declaration line that means we'll be coding in HTML 5.0. Following up are the ```<html>``` and ```<head>``` tags. Our html markup will be contained inside the two ```<html>``` tags. The ```<head>``` tag is where we will put some of our specifications which we normally don't want the end-user to see. The first ```<meta>``` tag tells the browser which character set to use. Here, it is set to UTF-8. The next ```<meta>``` tag specifies the page's dimensions and scaling. There is no problem in not understanding how these tags work, as they aren't very important. ```<title>``` is something that you should probably change. It is the text that is shown on the browser tab. Here it is set to repl.it. Coming next up in the ```<head>``` tag is ```<link>```. It tells the browser to load the CSS styling from the style.css file. It also informs the browser that it a css text file.
+
+Here the ```<head>``` tag ends, giving way to the body tag.  ```<body>``` is where we write the markup that the viewer will see. Repl.it has already created a ```<script>``` tag here that imports the JS code. Now that we have finished with understanding the boilerplate, let's start writing some of our own code.
+##### The HTML Markup
+As we can see from the final website gif, our site will have two parts, the first part which houses the buttons, and the second part, which houses the actual map. Let's get on with writing the code for the buttons. 
+
+We can use HTML default buttons, but they look like they should've been abandoned along with Windows 1999, so let's spice things up a little and design some buttons of our own. Above the ```<script>``` tag and below the opening ```<body>``` tag, add this.
+```html
+    <div class='first' style="margin-left:auto;margin-right:auto;">
```

Comment:
there is no need for indentation here since there is no parent element in this particular code block.